test/test_line.py

```diff
@@ -358,6 +358,44 @@ def test_meth_colliderect(self):
         with self.assertRaises(TypeError):
             A.colliderect(1, 5)
 
+    def test_meth_move(self):
```

Comment:
Should at least one integer be passed as well to test that? It doesn't "just" take floats, presumably.

Reply:
i dont really understand what you want to me to do and i would like you to elaborate.